app/src/Demo/Domain/Service/Administrator/SetActive.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Demo\Domain\Service\Administrator;
+
+use Demo\Domain\Model\Administrator\AdministratorDto;
+use Demo\Domain\Model\Administrator\AdministratorRepository;
+use Demo\Domain\Service\Administrator\SetActiveInterface;
+use Ivoz\Core\Domain\Service\EntityTools;
+
+class SetActive implements SetActiveInterface
+{
+    public function __construct(
+        private AdministratorRepository $administratorRepository,
+        private EntityTools $entityTools
+    ) {
+    }
+
+    public function execute(int $administratorId): void
+    {
+        $administrator = $this->administratorRepository->find($administratorId);
+
+        // I don’t think we should launch an exception in this case.
+        if ($administrator === null) {
+            return;
+        }
+
+        /** @var AdministratorDto $administratorDto */
+        $administratorDto = $this->entityTools->entityToDto($administrator);
+        $administratorDto->setActive(1);
+        $this->entityTools->persistDto($administratorDto);
```

Comment:
This will update entity state too:
```suggestion
        $this->entityTools->persistDto($administratorDto, $administrator);
```